### ISSUE TITLE
batch_size = 1

### ISSUE THREAD

**Issue by Soufiane-Fartit** [Soufiane-Fartit/autoencoder_reconstruction#1]
when I set the batch size to 1 the training goes without problems. But if I set it to a value higher than 1 it generates an error in the end of the first epoch during the validation.

InvalidArgumentError (see above for traceback): Incompatible shapes: [batch_size, 128,128] vs. [batch_size]

**Comment by Soufiane-Fartit** [comment]
[SOLVED]:
the validation set length should be a multiple of the batch_size

**Comment by Soufiane-Fartit** [comment]
[SOLVED]:
the validation set length should be a multiple of the batch_size
